Add taxonomy command

antgonza opened the pull request:
Depends on https://github.com/qiita-spots/qtp-diversity/pull/11; so first review/merge that one.

unique-identifier commented on qp_qiime2/__init__.py:
```diff
@@ -15,24 +18,40 @@
+qp_qiime2_dbs = glob(join(qp_qiime2_dbs, '*.qza'))
+if len(qp_qiime2_dbs) < 1:
+    raise ValueError(
+        "ENV QP_QIIME2_DBS points to a folder without qza's, please set.")
```
I believe qza's -> QZAs or QZA files.

antgonza replied:
K, thanks!

unique-identifier commented on qp_qiime2/__init__.py:
```diff
@@ -228,3 +243,71 @@
+        req_params[ename] = ('choice:[%s]' % qp_qiime2_dbs, default)
+        req_params['qp-hide-param' + ename] = ('string', pname)
+    else:
+        raise ValueError('Found non expected input: "%s", in '
```
non expected -> unexpected

antgonza replied:
k, thanks.

unique-identifier commented on qp_qiime2/tests/test_qiime2.py:
```diff
@@ -76,6 +79,114 @@ def test_not_analysis_artifact(self):
+            'qp-hide-plugin': 'feature-classifier',
+            'The feature data to be classified.': '8',
+            'qp-hide-paramDirection of reads with respect to reference '
+            'sequences. same will cause reads to be classified unchanged; '
```
same -> Same?

antgonza replied:
This comes from Q2, if you feel strongly about it, please open an issue directly in the plugin feature-classifier.

unique-identifier commented on qp_qiime2/tests/test_qiime2.py:
```diff
@@ -76,6 +79,114 @@ def test_not_analysis_artifact(self):
+            'used at all, which is useful for debugging. For n_jobs below -1, '
+            '(n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs '
+            'but one are used. (n_jobs)': '1', 'qp-hide-paramNumber of reads '
+            'to process in each batch. If 0, this parameter is autoscaled to '
```
autoscaled -> auto-scaled?

antgonza replied:
This comes from Q2, if you feel strongly about it, please open an issue directly in the plugin feature-classifier.

antgonza commented:
Thanks for review. Yes, we used to check for failure but as we changed the mapping file in Qiita now the command is actually successful so we changed the test.
